package.json

```diff
@@ -139,6 +139,14 @@
     "parse-url": "^8.1.0",
     "moment-timezone": "^0.5.35",
     "aws-cdk-lib": "~2.53.0",
-    "amplify-cli-core": "^4.0.0-beta.4"
+    "**/amplify-cli-core": "4.0.0-beta.5",
+    "**/@aws-amplify/amplify-environment-parameters": "1.3.4-beta.0",
+    "**/amplify-category-function": "4.3.2-beta.0",
+    "**/amplify-cli-logger": "1.2.4-beta.0",
+    "**/amplify-cli-shared-interfaces": "1.1.1-beta.3",
+    "**/amplify-function-plugin-interface": "1.9.6-beta.3",
+    "**/amplify-headless-interface": "1.17.1-beta.0",
+    "**/amplify-storage-simulator": "1.7.3-beta.0",
+    "**/amplify-util-headless-input": "1.9.10-beta.0"
```

Comment:
this locks packages that were incorrectly resolving to wrong tag.

Reply:
What's the `**/` prefix in package names?

Reply:
@phani-srikar it means apply to all dependency graph legs (fwiw previous definition seems to be no-op).
See https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/#toc-how-to-use-it .

Reply:
see also cli definitions for comparison 
https://github.com/aws-amplify/amplify-cli/blob/259f75c681bee9b26115eec2cc5b04dca7202c0a/package.json#L142-L151